pkgs/by-name/co/coyim/package.nix

```diff
@@ -24,16 +25,30 @@ buildGoModule {
 
   vendorHash = "sha256-zG7r/Db6XiwKoHRduGj3tEh/KT1hsuBoSGLYaZ+qO0Y=";
 
-  nativeBuildInputs = [ pkg-config wrapGAppsHook3 ];
+  nativeBuildInputs = [
+    pkg-config
+    wrapGAppsHook3
+  ];
 
-  buildInputs = [ glib cairo gdk-pixbuf gtk3 adwaita-icon-theme ];
+  buildInputs = [
+    glib
+    cairo
+    gdk-pixbuf
+    gtk3
+    adwaita-icon-theme
+  ];
+
+  patchPhase = ''
```

Comment:
```suggestion
  postPatch = ''
```

Reply:
Overriding `patchPhase` will ignore `patches`. Also you should add `runHook prePatch` and `runHook postPatch`.